src/sentry/utils/snuba.py

```diff
@@ -126,6 +126,12 @@ def raw_query(start, end, groupby=None, conditions=None, filter_keys=None,
     with timer('get_project_issues'):
         issues = get_project_issues(project_ids, filter_keys.get('issue')) if get_issues else None
 
+    if len(issues) == 1:
+        group_id = issues[0][0]
+        group = Group.objects.get(pk=group_id)
+        start = max(start, group.first_seen - timedelta(minutes=5))
+        end = min(end, group.last_seen + timedelta(minutes=5))
```

Comment:
I don't even know if we *need* to expand `start` but it seems harmless (we never go lower than the original `start`).

`end` needs to be padded by the worst-case-buffer-lag-time. In other words, if buffer (which writes out `Group` updates) is behind or stopped, this code would restrict queries against "times close to `now()`" because the `Group.last_seen` may be out of date in Postgres.

Honestly, since we're dealing with ClickHouse here maybe it should be 1 hour instead of 5 minutes.